### PR TITLE
initial version of plotclipper

### PR DESCRIPTION
**Pull request by max-zilla**
_please edit this template before submitting a new pull request_

## Checklist:

* [ ] I updated the documentation  
     
* [ ] Relevant tests (and test data) have been added or updated and they pass

### CODE REVIEW

**Comment by max-zilla** [comment]
@tcnichol FYI

**Comment by max-zilla** [comment]
before we call clip_raster we can use your function to check whether there is overlap:
https://github.com/terraref/terrautils/pull/62

File: extractors/plotclipper/terra_plotclipper.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        for filename in files_to_process:
+            self.log_info(resource, "Attempting to clip into plot shards")
+
+            for plotname in all_plots:
+                if plotname.find("KSU") > -1:
+                    continue
+
+                bounds = all_plots[plotname]
+                tuples = geojson_to_tuples_betydb(yaml.safe_load(bounds))
+
```
Comment (max-zilla) [inline extractors/plotclipper/terra_plotclipper.py]: basically here call your find_plots_intersect_boundingbox() function and only generate out_img and folders for the overlapping plots.

Comment (tcnichol) [reply extractors/plotclipper/terra_plotclipper.py]: removed those 2 lines

**Pull request merged**